Add plugin: Tab Navigator

o02c opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/o02c/obsidian-tab-navigator/tree/main

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[name: 'Search Tabs',](https://github.com/o02c/obsidian-tab-navigator/blob/f7d58e07a519d8d1c4579693c914d34e2d1ba657/src/main.ts#L10), [name: 'Remove Duplicate Tabs',](https://github.com/o02c/obsidian-tab-navigator/blob/f7d58e07a519d8d1c4579693c914d34e2d1ba657/src/main.ts#L40)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

[function isFileView(view: any): view is { file: any } {](https://github.com/o02c/obsidian-tab-navigator/blob/f7d58e07a519d8d1c4579693c914d34e2d1ba657/src/SearchModel.svelte#L116-L118)
This method should be replaced with this call:
```ts
if (leaf.view instanceof FileView)
```

o02c commented:
Thanks for the review. I fixed it in version [0.1.1](https://github.com/o02c/obsidian-tab-navigator/releases/tag/0.1.1):
- UI now uses sentence case: "Search tabs", "Remove duplicate tabs"
- Replaced `isFileView` with `if (leaf.view instanceof FileView)`,

aaronczichon commented on community-plugins.json:
```diff
@@ -11445,10 +11445,17 @@
+        "name": "Mapbox Location Image",
+        "author": "Aaron Czichon",
+        "description": "Renders an mapbox location image based on given coordinates.",
+        "repo": "aa}ronczichon/obsidian-location-plugin"
```
@joethei I think there was a merge error here in this PR :) 
Previously it was `"repo": "aaronczichon/obsidian-location-plugin"` and here it is `"repo": "aa}ronczichon/obsidian-location-plugin"`

aaronczichon replied:
Fixed it here #3429
